cluster/manifests/api-monitoring-controller/configmap.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: skipperdefaultfilters
```

Comment:
Some dashes in the name, please? :)

Reply:
Somehow it didn't work in our cluster and I had to delete dashes... Should we try with dashes anyway?

Reply:
It should work fine, what issues did you have?

Reply:
I don't remember the exact error message, but it was related to the name (I had underscores or dashes within the name). Let's try it with dashes, I'm happy to revert the change if it doesn't work.